mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAclModule.java

```diff
@@ -42,6 +44,17 @@
             .addPartitionKey(CassandraACLTable.ID, timeuuid())
             .addColumn(CassandraACLTable.ACL, text())
             .addColumn(CassandraACLTable.VERSION, bigint()))
+
+        .table(CassandraACLV2Table.TABLE_NAME)
+        .comment("Holds mailbox ACLs. This table do not rely on a JSON representation nor on LWT, contrary to the acl table it replaces.")
```

Comment:
could you describe what it does (positive description, not description in opposition to previous one) ?